src/sentry/tasks/post_process.py

```diff
@@ -1440,6 +1441,7 @@ def detect_new_escalation(job: PostProcessJob):
                 # TODO(snigdha): reuse manage_issue_states when we allow escalating from other statuses
                 add_group_to_inbox(group, GroupInboxReason.ESCALATING)
                 record_group_history(group, GroupHistoryStatus.ESCALATING)
+                auto_update_priority(group, PriorityChangeReason.ESCALATING)
```

Comment:
Actually one more thing here. I think we want to move the call here below the Activity object below so that the entries go "Sentry marked this issue as escalating" -> "Sentry marked this issue as high priority because it escalated"

Reply:
wrote it in the PR description and still got it wrong 😅